app/src/main/java/com/myfitnesstracker/ui/main/MainFragment.kt

```diff
@@ -46,11 +46,12 @@ class MainFragment : Fragment() {
     }
 
     fun saveBMI(bmi: String): String {
-           var weight = txtWeight.text.toString().toDouble()
-           var height = txtHeight.text.toString().toDouble()
-           var bmi = (weight / (height * height)) * 703
 
-            return "$bmi"
+        var weight = txtWeight.text.toString().toDouble()
+        var height = txtHeight.text.toString().toDouble()
+        var bmi = (weight / (height * height)) * 703
+
+        return "$bmi"
```

Comment:
Reduced the Indent to 4 spaces.
According to: https://kotlinlang.org/docs/reference/coding-conventions.html